### PR TITLE
Export data to excel

### PR DESCRIPTION
**Pull request by AndreyGoranov**
Issue: https://github.com/podkrepi-bg/frontend/issues/700

- Add Export module. 
- Export service to be used around the app in different services that would require exporting.
- There is an option to specify everything about the template in a single JSON or you could use a default setup with all columns from the data provided for exporting.
- Add new dependency: https://github.com/exceljs/exceljs
- Use types for the exportable data and add types for the template as well

### CODE REVIEW

**Review by kachar — COMMENTED** [review]
@AndreyGoranov Everything looks smooth and nice!

The only thing that looks strange to me is the 15k rows diff by the added `yarn.lock` file. Looks like it's been changed entirely. Can you check that please?

**Comment by AndreyGoranov** [comment]
@kachar yes will do I had a merge conflict on it and probably I didn't resolve it right

**Comment by kachar** [comment]
@AndreyGoranov You can delete the local file and run `yarn` to regenerate it.

I hope you're using yarn 3 (`yarn set version berry`)

**Comment by AndreyGoranov** [comment]
@kachar I think its ok now ?

File: apps/api/src/export/export.service.ts [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import { Injectable } from '@nestjs/common'
+import { PrismaService } from '../prisma/prisma.service'
+import { Response } from 'express'
+import { createWorkbook } from './helpers/createWorkbook'
+import { ExportableData } from '@prisma/client'
```
Comment (kachar) [inline apps/api/src/export/export.service.ts:5]: @AndreyGoranov Can you help me understand what is `ExportableData` as I can find an update of `schema.prisma` and this type is not being generated.

Comment (AndreyGoranov) [reply apps/api/src/export/export.service.ts:5]: It's just a Type:  export Type ExportableData = Donations[] | Campaigns[]

**Review by kachar — APPROVED** [review]
@AndreyGoranov Nice upgrade of the api!

It wasn't easy to make the tests and check green, but we've managed it. Thanks for the collaboration along the way

**Review by igoychev — COMMENTED** [review]
Change requested: 
The amounts in the exported file should be divided by 100 to contain the cents.

**Comment by kachar** [comment]
@AndreyGoranov Can you please create a PR with the update that @igoychev mentioned above?

**Comment by AndreyGoranov** [comment]
https://github.com/podkrepi-bg/api/pull/396

**Pull request merged**